cirq/google/api/v2/run_context.proto

```diff
@@ -12,6 +12,19 @@ message RunContext {
   repeated ParameterSweep parameter_sweeps = 1;
 }
 
+// A batch of contexts for running a bundled batch of programs
+// To be used in conjunction with BatchedProgram
+message BatchedRunContext {
```

Comment:
You might consider moving all these into a new `batch.proto` file, so they are closer together and the relationship between them should be easier to see:
```python
# batch.proto

message BatchProgram { ... }
message BatchRunContext { ... }
message BatchResult { ... }
```

Reply:
Done.